feature-tests/Dockerfile

```diff
@@ -10,19 +10,20 @@ WORKDIR /
 # Set up AWS CLI
 RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
     unzip awscliv2.zip && \
-	./aws/install
+    ./aws/install
 
 # COPY Files from src directory
 COPY package.json /package.json
 COPY yarn.lock /yarn.lock
 COPY tsconfig.json /tsconfig.json
 COPY jest.config.ts /jest.config.ts
-COPY jest-cucumber-config.js /jest-cucumber-config.js
+# COPY jest-cucumber-config.js /jest-cucumber-config.js
```

Comment:
uncomment as the file is referenced in tsconfig.json

Reply:
Resolved on latest Commit, thanks Taher